jina/types/document/__init__.py

```diff
@@ -186,9 +187,16 @@ def modality(self, value: str):
     def content_hash(self):
         return self._document.content_hash
 
-    def update_content_hash(self):
-        """Update the document hash according to its content."""
-        self._document.content_hash = get_content_hash(self._document)
+    def update_content_hash(self, mask: Tuple[str] = ('id', 'chunks', 'matches', 'content_hash')) -> None:
```

Comment:
I opened the issue to have exactly the opposite logic, I want the mask to be the fields that we want to include and not the ones we want to exclude.

Reply:
> I opened the issue to have exactly the opposite logic, I want the mask to be the fields that we want to include and not the ones we want to exclude.

I also think this is more resilient to any future adding of new fields